$NetBSD: patch-ak,v 1.1 2002/09/15 06:51:38 kim Exp $

The configure.in patch manually applied to configure, as
our autoreconf produces an extensively different result.

--- configure.orig	Thu Feb 14 08:11:21 2002
+++ configure	Sat Sep 14 23:44:07 2002
@@ -4077,9 +4077,15 @@
 
 #include <time.h> /* to get time_t on SCO */
 #include <sys/types.h>
-#if defined(SVR4) && !defined(DGUX)
+#include <sys/param.h>
+#if (defined(SVR4) && !defined(DGUX)) \
+ || (defined(__NetBSD_Version__) && (__NetBSD_Version__ >= 106050000))
 #include <utmpx.h>
 #define utmp utmpx
+# ifdef __NetBSD__
+#  define pututline pututxline
+#  define getutent  getutxent
+# endif
 #else
 #include <utmp.h>
 #endif
